CODING_STYLE.md

```diff
@@ -134,7 +133,7 @@ int i = 0;
 int j = 0;
 char* s = nullptr;
 MeanAndSigma ms meanAndSigma(std::vector<float> const& _v, Accuracy _a);
-Derived* x = dynamic_cast<Derived*>(base);
+auto x = dynamic_cast<Derived*>(base);
```

Comment:
You could think about `auto` vs `auto*` here. Doesn't make a difference in this example, but if you wanted to, say, const-qualify the pointer then only the latter gives that flexibility.

Reply:
Yeah, I thought about it and actually *do* prefer pointers to generally remain visible myself - I changed it.